docs/windows.rst

```diff
@@ -88,7 +88,7 @@ MSYS2 is a more maintained version of MSYS, but specially geared towards MinGW-w
 
 **Cygwin**
 
-Cygwin provides ports of many GNU/Linux tools and a POSIX API layer. This is the most complete way to get the GNU/Linux terminal feel under Windows. Cygwin has a setup that helps you install all the tools you need. Drawback of Cygwin is that the POSIX API layer (cygwin1.dll) is not fast.
+Cygwin provides ports of many GNU/Linux tools and a POSIX API layer. This is the most complete way to get the GNU/Linux terminal feel under Windows. Cygwin has a setup that helps you install all the tools you need. A drawback of Cygwin is that the POSIX API layer (cygwin1.dll) is not fast.
```

Comment:
Last sentence could read "One drawback of Cygwin is that it has poor performance."